src/components/__tests__/__snapshots__/Leadership.test.jsx.snap

```diff
@@ -0,0 +1,578 @@
+          <img
+            src="jordan.png"
+          />
+          <p
+            class="color_black is-bold is-family-secondary"
+            data-testid="name"
+          >
+            Jordan Christensen
+          </p>
+          <p
+            class="color_chalice"
+            data-testid="role"
+          >
+            Mobile Engineer
+          </p>
+        </div>
+        <div
+          class="card"
+        >
+          <img
+            src="justin.jpg"
+          />
+          <p
+            class="color_black is-bold is-family-secondary"
+            data-testid="name"
+          >
+            Justin Gent
+          </p>
+          <p
+            class="color_chalice"
+            data-testid="role"
+          >
+            Mobile Engineer
+          </p>
+        </div>
+        <div
+          class="card"
+        >
+          <img
+            src="lowell.jpg"
+          />
+          <p
+            class="color_black is-bold is-family-secondary"
+            data-testid="name"
+          >
+            Lowell Jacobs
+          </p>
+          <p
+            class="color_chalice"
+            data-testid="role"
+          >
+            Engineer
+          </p>
+        </div>
+      </section>
+    </div>
+  </div>,
+  "debug": [Function],
+  "findAllByAltText": [Function],
+  "findAllByDisplayValue": [Function],
+  "findAllByLabelText": [Function],
+  "findAllByPlaceholderText": [Function],
+  "findAllByRole": [Function],
+  "findAllByTestId": [Function],
+  "findAllByText": [Function],
+  "findAllByTitle": [Function],
+  "findByAltText": [Function],
+  "findByDisplayValue": [Function],
+  "findByLabelText": [Function],
+  "findByPlaceholderText": [Function],
+  "findByRole": [Function],
+  "findByTestId": [Function],
+  "findByText": [Function],
+  "findByTitle": [Function],
+  "getAllByAltText": [Function],
+  "getAllByDisplayValue": [Function],
+  "getAllByLabelText": [Function],
+  "getAllByPlaceholderText": [Function],
+  "getAllByRole": [Function],
+  "getAllByTestId": [Function],
+  "getAllByText": [Function],
+  "getAllByTitle": [Function],
+  "getByAltText": [Function],
+  "getByDisplayValue": [Function],
+  "getByLabelText": [Function],
+  "getByPlaceholderText": [Function],
+  "getByRole": [Function],
+  "getByTestId": [Function],
+  "getByText": [Function],
+  "getByTitle": [Function],
+  "queryAllByAltText": [Function],
+  "queryAllByDisplayValue": [Function],
+  "queryAllByLabelText": [Function],
+  "queryAllByPlaceholderText": [Function],
+  "queryAllByRole": [Function],
+  "queryAllByTestId": [Function],
+  "queryAllByText": [Function],
+  "queryAllByTitle": [Function],
+  "queryByAltText": [Function],
+  "queryByDisplayValue": [Function],
+  "queryByLabelText": [Function],
+  "queryByPlaceholderText": [Function],
+  "queryByRole": [Function],
+  "queryByTestId": [Function],
+  "queryByText": [Function],
+  "queryByTitle": [Function],
+  "rerender": [Function],
+  "unmount": [Function],
+}
+`;
```

Comment:
We should probably talk about changing the about us page to include us. I'm glad the test works though!